docs/source/install/monitoring_stack.rst

```diff
@@ -10,12 +10,12 @@ For evaluation, you can run Scylla Monitoring Stack on any server (or laptop) th
 
 .. include:: min-prod-hw.rst
 
-.. _`Scylla Prometheus API`: monitoring_apis#prometheus
+.. _`Scylla Prometheus API`: /reference/monitoring_apis#prometheus
```

Comment:
Relative URLs should not start with / (slash).

```suggestion
```` 

Suggestion: Delete this line, and use the ``:doc:`` directive instead.